solr/core/src/java/org/apache/solr/filestore/DistribFileStore.java

```diff
@@ -534,7 +534,6 @@ public void refresh(String path) {
         @SuppressWarnings({"rawtypes"})
         List myFiles = list(path, s -> true);
         for (Object f : l) {
-          // TODO: https://issues.apache.org/jira/browse/SOLR-15426
```

Comment:
@madrob you opened up this JIRA, and I *think* this TODO is incorrect or otherwise overcome by events TODO.  Regardless, it doesn't appear connected to SOLR-15426...   Can you confirm?

Reply:
I think it was actually related to SOLR-15385, I must have made a copy/paste error here. Reading my comments and surface-level tracing the code suggests that this is still an issue.

Reply:
I think what I am hearing is that I should update the JIRA that we are pointing to, but otherwise KEEP the todo.